x-pack/plugins/security_solution/public/cases/pages/index.tsx

```diff
@@ -71,6 +60,22 @@ const CaseContainerComponent: React.FC = () => {
     [openFlyout, telemetry]
   );
 
+  const onRuleDetailsClick = useCallback(
+    (ruleId: string | null | undefined) => {
+      if (ruleId) {
+        openFlyout({
+          right: {
+            id: RulePanelKey,
+            params: {
+              ruleId,
+            },
+          },
+        });
```

Comment:
NIT: this can be formatted in one line:
```suggestion
        openFlyout({ right: { id: RulePanelKey, params: { ruleId } } });
```